### PR TITLE
add support for redpallas

### PR DESCRIPTION
**Pull request by conradoplg**
Depends on #62 

This adds support for RedPallas using the `redpallas` feature (e.g. `cargo run --bin dkg --features redpallas`)

[Features should be additive](https://doc.rust-lang.org/cargo/reference/features.html#feature-unification) and this goes against that, but since we have only binaries and not libraries, it seems fine.

When  `redpallas` is activate, support for rerandomized FROST is also enabled. Currently the randomizer is always read (even though ideally it should be generated by the coordinator) because this is currently required by the demo.

### CODE REVIEW

File: participant/src/round2.rs [hunk 1]
```diff
@@ -31,6 +37,28 @@ pub fn round_2_request_inputs(
...
 
+    #[cfg(feature = "redpallas")]
+    {
+        logger.log("Enter the randomizer (hex string):".to_string());
+
+        let mut json = String::new();
+
+        input.read_line(&mut json).unwrap();
+
+        // TODO: fix error
```
Comment (natalieesk) [inline participant/src/round2.rs:48]: Does this need to be fixed as part of this story?

**Review by natalieesk — COMMENTED** [review]
It looks good. Just one question.

File: participant/src/round2.rs (hunk 1 above)
Comment (conradoplg) [reply participant/src/round2.rs]: thanks! forgot about it, fixed in 1e17731

**Review by natalieesk — APPROVED** [review]
Looks good

**Pull request merged**